schema-sandbox/schema.json

```
@@ -0,0 +1,69 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "http://citation-file-format.github.io/1.1.0/schema",
+    "title": "Citation File Format",
+    "description": "A file with citation metadata for software or datasets",
+    "additionalProperties": false,
+    "type": "object",
+    "properties": {
+        "cff-version": {
+            "type": "string",
+            "pattern": "^1\\.1\\.0$",
+            "description": "The version of CFF used for providing the citation metadata"
+        },
+        "message": {
+            "type": "string",
+            "description": "A message to the (human) reader of the file to let them know what to do with the citation metadata"
+        },
+        "type": {
+            "type": "string",
+            "enum": [
+                "dataset",
+                "software"
+            ]
+        },
+        "abstract": {
+            "type": "string"
+        },
+        "authors": {
+            "type": "array",
+            "minItems": 1,
+            "items": {
+                "anyOf": [
+                    {"$ref": "#definitions/person"},
+                    {"$ref": "#definitions/entity"}
```

```suggestion
                    {"$ref": "#/definitions/person"},
                    {"$ref": "#/definitions/entity"}
```
probably needs slashes

---

Does without, but if it works with them, I'd prefer it with them... I'll try soon as I can.